test/relations.test.js

```diff
@@ -1349,47 +1450,25 @@ describe('relations', function() {
       });
 
       it('can determine the collect via model name', function() {
-        var physician = new Physician({id: 1});
+        var physician = new Physician({id: idPhysician});
         var scope1 = physician.xxx._scope;
         scope1.should.have.property('collect', 'patient');
         scope1.should.have.property('include', 'patient');
       });
 
       it('can determine the collect via keyThrough', function() {
-        var patient = new Patient({id: 1});
+        var patient = new Patient({id: idPatient});
         var scope2 = patient.yyy._scope;
         scope2.should.have.property('collect', 'foo');
         scope2.should.have.property('include', 'foo');
       });
     });
   });
 
-  describe('hasMany through - customized relation name and foreign key', function() {
```

Comment:
Why was this removed? Please bring it back, use `bdd.describeIf` to skip it for Cassandra (if needed).